assets/sass/_staticman.sass

```diff
@@ -0,0 +1,95 @@
+.page__comments
+  /* HTML form */
+  .page__comments-form
+    margin-top: 1rem
+    &.disabled
+      input, button, textarea, label
+        pointer-events: none
+        cursor: not-allowed
+        filter: alpha(opacity = 65)
+        box-shadow: none
+        opacity: 0.65
+
+    .form-group
+      margin: 0.6rem 0 0.6rem 0
+      textarea
+        width: 100%
+      textarea, input
+        padding: 0.5rem
+        border: 1px solid $secondary
+        background: $background
+        color: $body-color
+        border-radius: 4px
+
+    .reply-notice
+      .reply-arrow
+        vertical-align: middle
+      .comment__avatar
+        display: inline-block
+        float: none
+        width: 2rem
+        height: 2rem
+        margin: 0 .6rem
+        vertical-align: text-bottom
+        border-radius: 50%
+      .close-btn
+        margin: 0 .6rem
+        &:after
+          content: '\2716'
+
+    .required
+      font-weight: bold
+
+    button
+      border-radius: 4px
+      background: $secondary
+      border: 1px solid $secondary
+      padding: 0.5rem
+      margin-right: 0.6rem
+
+  /* Static comments */
+  $avatar__size: 5rem
+  $avatar__sep: 3rem
+  $reply__indent: 2rem
```

Comment:
Feel free to change the values here. Since I'm not a web designer, I haven't considered the mobile loading issue.